📝 Add more installation methods (#43)
ajnart/homarr-docs: docs/introduction/installation.md

After you've installed Homarr, you can find it under the tab "Docker".

The official support thread is located [here](https://forums.unraid.net/topic/123478-support-smartphonelover-homarr/), but we prefer to communicate over GitHub and Discord instead.

---

## Installation on Synology

> The installation process is also quite easy and fast on Synology devices.
> Since this method is used less often, we recommend you the thiry party guide of mariushosting.
> 
> https://mariushosting.com/how-to-install-homarr-on-your-synology-nas/
> 
> :::note
> 
> This is a third party guide - we do not garantuee if it's up to date nor offer support. Use at your own risk.
> 
> :::

Reviewer: ```suggestion
The installation process is quite easy and fast on Synology devices.
Since this method is used less often, we recommend you to follow this guide written by mariushosting.

https://mariushosting.com/how-to-install-homarr-on-your-synology-nas/

:::note

This is a third party guide! It might not be up-to-date, nor do we offer official support.

:::
```

## Installation on QNAP

> We recommend you to install Homarr via Cointainer Station.
> 
> You can also use this: https://www.myqnap.org/product/homarr/
> 
> A Chinese tutorial for this can also be found here: https://post.smzdm.com/p/awzm7op2/
> 
> :::caution
> 
> This is an unvalidated third party guide - we do not garantuee if it's up to date nor offer support. Use at your own risk.
> 
> :::
> 

Reviewer: ```suggestion
When installing on QNAP, we recommend you to install Homarr using [Container Station]( https://www.qnap.com/en/software/container-station) or [this third-party image](https://www.myqnap.org/product/homarr/).

There is also a guide available, but note that it is written in Chinese: https://post.smzdm.com/p/awzm7op2/

:::caution

This is a *unvalidated* third party guide! It might not be up-to-date, nor do we offer official support.

:::

```

## Installation using Saltbox

> You can also use this to install on Saltbox:

Reviewer: Empty line?

> :::caution
> 
> This is an unvalidated third party guide - we do not garantuee if it's up to date nor offer support. Use at your own risk.
> 
> :::

Reviewer: ```suggestion
:::caution

This is a *unvalidated* third party guide! It might not be up-to-date, nor do we offer official support.

:::
```

</Tabs>

---

> ## Anything else in mind?
> Have you installed Homarr using a different method?
> It would be amazing if you could share, so other users can install Homarr too!

Reviewer: ```suggestion
## Help us add more installation methods!
Have you installed Homarr using a different method? Please share it with us, so we can expand to a bigger userbase!
```

> ## Troubleshooting

Reviewer: Please add these lines back.
